src/styles/deleteAccount.module.css

```diff
@@ -19,30 +19,30 @@
     background-color: var(--main-color);
   }
   50% {
-     color: var(--main-color);
-     background-color: red;
+    color: var(--main-color);
+    background-color: red;
   }
-  100%{
-     color: red;
-     background-color: var(--main-color); 
+  100% {
+    color: red;
+    background-color: var(--main-color);
   }
 }
-.h2Alert{
+.h2Alert {
   color: red;
 }
-#deleteQuestion{
+#deleteQuestion {
   display: flex;
   flex-direction: row;
   gap: 15px;
   justify-content: center;
 }
-#btnConfirmDelete{
+#btnConfirmDelete {
   background-color: red;
 }
-.yesNo{
+.yesNo {
   border: none;
```

Comment:
YesNo is not a good name, if I want to know what this is doing I need to check where its being used in your code. 

Change this into something that we can understand from the name only